test-cases/README.md

```diff
@@ -167,45 +174,100 @@ This command will read all test results from Jira, and upload them to Polarion.
 JIRA_USERNAME=ju JIRA_PASSWORD=jp POLARION_USERNAME=pu POLARION_PASSWORD=pp ./tools.sh polarion testrun --epic INTLY-5390
 ```
 
-## Tags
+## Test Case Metadata
 
-Tags can be added for each test case inside the metadata section in the header. Tags can be defined globally
-or specifically for each variant.
+## Environments
 
-Supported tags:
+The environment field specify against wich environment/setup the test need to be executed.
```

Comment:
Should we have a tag for private cluster also here?

Reply:
yes is the `osd-private-post-upgrade` environemnt